site_config/docs2-0-0.js

```diff
@@ -283,7 +283,7 @@ export default {
         children: [
           {
             title: '简介',
-            link: '/zh-cn/docs/About_DolphinScheduler/About_DolphinScheduler.html',
+            link: '/zh-cn/docs/2.0.1/user_doc/About_DolphinScheduler/About_DolphinScheduler.html',
```

Comment:
I fond out we still have this file in https://github.com/apache/dolphinscheduler-website/blob/5f4d4a913d0909f36a33c776c1d9cef3adb1ba11/docs/zh-cn/About_DolphinScheduler/About_DolphinScheduler.md could you remove the file. and add it to each verison doc?